foreman/foreman.spec

```diff
@@ -199,6 +199,8 @@ BuildRequires: %{?scl_prefix}rubygem(autoprefixer-rails) >= 5.2
 BuildRequires: %{?scl_prefix}rubygem(autoprefixer-rails) < 6.0
 BuildRequires: %{?scl_prefix}rubygem(patternfly-sass) >= 2.7
 BuildRequires: %{?scl_prefix}rubygem(patternfly-sass) < 3.0
+BuildRequires: %{?scl_prefix}rubygem(sprockets) >= 3
+BuildRequires: %{?scl_prefix}rubygem(sprockets) < 4
```

Comment:
I'd call it a mistake on the Gemfile. However, I thought ~> 3 was equivalent to ~> 3.0. If I change it in Foreman to `~> 2` (*) , it installs `2.12.4` instead of `3.5.2`. Are you sure of that?

\* it doesn't actually work with foreman, just for the sake of testing